docs/topics/request-response.rst

```diff
@@ -675,6 +680,14 @@ Response objects
         they're shown on the string representation of the Response (`__str__`
         method) which is used by the engine for logging.
 
+    .. attribute:: Response.address
+
+        A Twisted object containing information about the server IP address.
+        See :class:`twisted.internet.address.IPv4Address` for additional information.
```

Comment:
It could be nice to avoid exposing twisted-specific structures, in case we move away from Twisted